specification/gedcom.md

```diff
@@ -580,6 +580,65 @@ In particular, those supporting extensions should keep in mind the following:
 
     </div>
 
+### Identifying structure types
+
+If a structure's tag is a documented extension tag, its structure type is defined by the URI given for that tag in the schema structure.
+
+If a structure's tag is an undocumented extension tag, its structure type is defined by that tag.
+
+If a structure's tag is a standard tag, its structure type depends on its superstructure type as follows:
+
+If the structure is a record with no superstructure, or if its superstructure type is defined in this specification; and if this specification lists a structure with that superstructure and that tag; then the structure type is given by this specification.
+
+If the structure's superstructure type is not defined by this specification and this specification does define a structure type whose URI is the concatenation of `https://gedcom.io/terms/v7/` and the tag, then the structure is an extended-use standard structure and its structure type is given by that URI.
+
+Otherwise, no structure with that tag is permitted in that context and the dataset is invalid (but may be valid under a future minor version; see [A Guide to Version Numbers] for more).
```

Comment:
```suggestion
Otherwise, no structure with that tag is permitted in that context and the dataset is invalid (but may be valid under a future minor version; see [A Guide to Version Numbers](#a-guide-to-version-numbers-unnumbered) for more).
```

Reply:
Because github and pandoc handle internal hyperlinks differently, I'd rather leave it as just `[A Guide to Version Numbers]`. We could use `a-guide-to-version-numbers` or add a custom section ID and it would work on gedcom.io but be a dead link in github; or add `a-guide-to-version-numbers-unnumbered` and it would be a dead link on gedcom.io but work in github; but I think the simple brackets are about as good as we can get in trying to make both readable.